test/workflow/module_tests/no_docker_test_cggnn.sh

```diff
@@ -1,5 +1,11 @@
+# Set up a virtual environment to run this test
+python -m venv env
+source env/bin/activate
+pip install ../..[workflow]
```

Comment:
We really shouldn't be creating inception virtual environments... installing from source then abandoning it.
The spt command should clearly be run in the development image, just like in all the other tests.
The nextflow run is what needs to be run outside, and now we have a new development environment dependency: nextflow. Previously nextflow was entirely containerized, but for this specific test we aren't doing that.

Reply:
(The install of spatialprofilingtoolbox and extras doesn't properly belong to the test functionality and wastes time.)